ufm-test-services/grafana/dashboards/metamask.json

```diff
@@ -0,0 +1,182 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "grafana",
+          "uid": "-- Grafana --"
+        },
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 0,
+  "links": [],
+  "liveNow": false,
+  "panels": [
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "PBFA97CFB590B2093"
```

Comment:
Should we put those UID into a const file , perhaps as enums so that we can pull from there ? I see we already use it multiple place in this file and that could be a source of misconfuguration?

Reply:
I setup the dashboard using Grafana and this file is an export of that dashboard - it shouldn't need to be modified by hand, and is probably better suited to be managed within Grafana's UI if changes need to be made

Specifically for this `datasource`, Grafana is being configured to use it by [this](https://github.com/ethereum-optimism/optimism/blob/wyatt/ufm/init-mm-metrics/ufm-test-services/grafana/provisioning/datasources/datasources.yml) file and [docker-compose.yml](https://github.com/ethereum-optimism/optimism/blob/933bedb46842584a969db4832bf95272dc1a2760/ufm-test-services/docker-compose.yml#L38), so the `uid` should always be in-sync since it's managed by these processes. I've tested completely tearing down the system and re-building the Docker images, and everything works fine with the `uid`s remaining static